README.md

```diff
@@ -1,69 +1 @@
-# :musical_note: JioSaavn API
```

Comment:
It was a preliminary act. It should be resolved by any moment.

Reply:
I advice you to merge this in another branch and host it to some subdomain. Then in future it might be updated for services using it right now.

Reply:
Approved and merged in `dev` branch

Reply:
Is the dev branch hosted on any subdomain like main branch is on saavn.me?

Reply:
as of now. but if you want to check you can use - jiosaavn-api-487dzmefk-sumitkolhe.vercel.app for now. can't gurantee till when this link will work

Reply:
I had to use this in https://github.com/blueedgetechno/windows11 , in the spotify application. I will wait for untill then :) .

Reply:
@blueedgetechno i've deployed it to a custom url for you. you can use https://saavn-dev.vercel.app/ this. This url will stay online so you need not worry

Reply:
Thanks

Reply:
For some reason, the /search api is only returing indian songs,
for example if I call https://dev.saavn.me/search?query=perfect , It doesn't return the Ed Sheeran song, Perfect
but If I directly call the jiosaavn endpoint used by /search endpoint, [api.php?__call=autocomplete.get&query=perfect](https://www.jiosaavn.com/api.php?__call=autocomplete.get&_format=json&_marker=0&includeMetaTags=1&query=perfect), It does return the song by Ed Sheeran.

I think it might be due to where the application is being hosted.

Reply:
Yes, you're correct. Jiosaavn has some kind of ip geolocation checks which provides songs based on location. Because of the APIs hosting location it doesn't give the exact songs because of the location.

Reply:
More strangely, they dont even show the song even if I search 'perfect by ed sheeran'. And where is it hosted?

Reply:
It's hosted on vercel.com

Other way around for this would be to use direct song links to get such songs.
Ex - https://saavn.me/song?link=https://www.jiosaavn.com/song/perfect/RgdTexthD1E